num_enum_derive/Cargo.toml

```diff
@@ -33,7 +33,7 @@ features = ["external_doc"]
 
 [dependencies]
 proc-macro2 = "1.0.60"
-proc-macro-crate = { version = "1", optional = true }
+proc-macro-crate = { version = "2.0.0", optional = true }
```

Comment:
It may be possible to use `>=1,<=2` to keep the MSRV the same and the old deps for people who need it (since I see no code changes, I suppose this will work fine)

Reply:
Seems reasonable to me! I can test it tomorrow, just to be sure.